tools/openblas_support.py

```diff
@@ -15,8 +15,9 @@
 
 OPENBLAS_V = '0.3.23.dev'
 OPENBLAS_LONG = 'v0.3.23-246-g3d31191b'
-BASE_LOC = 'https://anaconda.org/multibuild-wheels-staging/openblas-libs'
-BASEURL = f'{BASE_LOC}/{OPENBLAS_LONG}/download'
+BASE_LOC = (
+    'https://anaconda.org/scientific-python-nightly-wheels/openblas-libs'
+)
```

Comment:
`BASE_LOC`  should be different depending on whether or not the weekly is used. If the weekly isn't being used, the url should still be `'https://anaconda.org/multibuild-wheels-staging/openblas-libs'`. 
 
In https://github.com/scipy/scipy/pull/18942 `base_loc` as a keyword arg to `download_openblas`. See [here](https://github.com/scipy/scipy/blob/9300bee6608a637b3a16b9ef5fe14256cd9a99fd/tools/openblas_support.py#L18-L21).

Reply:
> If the weekly isn't being used, the url should still be 'https://anaconda.org/multibuild-wheels-staging/openblas-libs'

I think this is because the tag needed by SciPy, `v0.3.20-571-g3dec11c6` has not been uploaded to https://anaconda.org/scientific-python-nightly-wheels/openblas-libs. But for NumPy, the tag `v0.3.23-246-g3d31191b` __has__ been uploaded to the new location, so here we can use the single site.

Reply:
ahh, the travis job did not upload, so indeed the new site is missing the aarch64 tarballs.

Reply:
I see. I think I misunderstood. From the name, I thought that `scientific-python-nightly-wheels` is only for nightlies and that tagged releases are still being uploaded to the old location.

Reply:
I restarted the travis job, let's see if this time it succeeds (the token was missing).

Reply:
So yeah to emphasise, `scientific-python-nightly-wheels/openblas-libs` is now holding the stable and weekly builds.